docs/resize-rbd.md

```diff
@@ -0,0 +1,277 @@
+    creationTimestamp: "2019-12-19T05:44:45Z"
+    finalizers:
+    - kubernetes.io/pvc-protection
+    name: rbd-pvc
+    namespace: default
+    resourceVersion: "3557"
+    selfLink: /api/v1/namespaces/default/persistentvolumeclaims/rbd-pvc
+    uid: efe688d6-a420-4041-900e-c5e19fd73ebf
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+    storageClassName: csi-rbd-sc
+    volumeMode: Filesystem
+    volumeName: pvc-efe688d6-a420-4041-900e-c5e19fd73ebf
+  status:
+    accessModes:
+    - ReadWriteOnce
+    capacity:
+      storage: 1Gi
+    phase: Bound
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+```
+
+Directory size in  pod using this PVC
+
+```
+kubectl exec -it csi-rbd-demo-pod sh
+# df -h
+Filesystem      Size  Used Avail Use% Mounted on
+overlay          40G  8.4G   29G  23% /
+tmpfs           1.9G     0  1.9G   0% /dev
+tmpfs           1.9G     0  1.9G   0% /sys/fs/cgroup
+/dev/vda1        40G  8.4G   29G  23% /etc/hosts
+shm              64M     0   64M   0% /dev/shm
+/dev/rbd0       976M  2.6M  958M   1% /var/lib/www/html
+tmpfs           1.9G   12K  1.9G   1% /run/secrets/kubernetes.io/serviceaccount
+tmpfs           1.9G     0  1.9G   0% /proc/acpi
+tmpfs           1.9G     0  1.9G   0% /proc/scsi
+tmpfs           1.9G     0  1.9G   0% /sys/firmware
+#
+
+```
+
+Now resize the PVC by editing the PVC (pvc.spec.resource.requests.storage)
+
+```bash
+#kubectl edit pvc rbd-pvc
+```
+
+Check PVC status after  editing the pvc storage
+
+```console
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      pv.kubernetes.io/bind-completed: "yes"
+      pv.kubernetes.io/bound-by-controller: "yes"
+      volume.beta.kubernetes.io/storage-provisioner: rbd.csi.ceph.com
+    creationTimestamp: "2019-12-19T05:44:45Z"
+    finalizers:
+    - kubernetes.io/pvc-protection
+    name: rbd-pvc
+    namespace: default
+    resourceVersion: "4773"
+    selfLink: /api/v1/namespaces/default/persistentvolumeclaims/rbd-pvc
+    uid: efe688d6-a420-4041-900e-c5e19fd73ebf
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 10Gi
+    storageClassName: csi-rbd-sc
+    volumeMode: Filesystem
+    volumeName: pvc-efe688d6-a420-4041-900e-c5e19fd73ebf
+  status:
+    accessModes:
+    - ReadWriteOnce
+    capacity:
+      storage: 1Gi
+    conditions:
+    - lastProbeTime: null
+      lastTransitionTime: "2019-12-19T05:49:39Z"
+      message: Waiting for user to (re-)start a pod to finish file system resize of
+        volume on node.
+      status: "True"
+      type: FileSystemResizePending
+    phase: Bound
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+
+```
+
+Now you  can see the pvc stats as FileSystemResizePending, once the kubelet
+calls the NodeExpandVolume to resize the PVC  on node, the status conditions
```

Comment:
change `the status conditions  will be updated and status will be updated` **to** `the status conditions
will be updated and `status.capacity.storage` will be updated`